administrator/language/de-DE/com_templates.ini

```
@@ -148,10 +148,8 @@ COM_TEMPLATES_IMAGE_WIDTH="Breite"
 COM_TEMPLATES_INVALID_FILE_NAME="Ungültiger Dateiname. Einen Dateinamen auswählen, der nur aus den folgenden Zeichen besteht: a-z, A-Z, 0-9, - und _."
 COM_TEMPLATES_INVALID_FILE_TYPE="Die Datei konnte nicht ausgewählt werden."
 COM_TEMPLATES_INVALID_FOLDER_NAME="Ungültiger Verzeichnisname. Einen Dateinamen auswählen, der nur aus den folgenden Zeichen besteht: a-z, A-Z, 0-9, - und _."
-COM_TEMPLATES_LAYOUTS_DIFFVIEW_HIDE_CORE="Original ausblenden"
-COM_TEMPLATES_LAYOUTS_DIFFVIEW_HIDE_DIFF="Unterschiede ausblenden"
-COM_TEMPLATES_LAYOUTS_DIFFVIEW_SHOW_CORE="Original anzeigen"
-COM_TEMPLATES_LAYOUTS_DIFFVIEW_SHOW_DIFF="Unterschiede anzeigen"
+COM_TEMPLATES_LAYOUTS_DIFFVIEW_CORE="Originaldatei"
+COM_TEMPLATES_LAYOUTS_DIFFVIEW_DIFF="Unterschiede"
```

Ggfs. Ist Änderungen besser?

```suggestion
COM_TEMPLATES_LAYOUTS_DIFFVIEW_DIFF="Änderungen"
```